plume/script/DeployDevnetContracts.s.sol

```diff
@@ -9,12 +9,12 @@ import { FaucetProxy } from "../src/proxy/FaucetProxy.sol";
 
 contract DeployDevnetContracts is Script {
 
-    address private constant FAUCET_ADMIN_ADDRESS = 0x7bFf5f044b3153E36e4D97944aB7f7Db0f2a761C;
-    address private constant ETH_ADDRESS = address(1);
+    address private constant FAUCET_ADMIN_ADDRESS = 0xC0A7a3AD0e5A53cEF42AB622381D0b27969c4ab5;
```

Comment:
Why did you change the address - the old admin address is the one we have access to